src/storage/db_interface_frontend.py

```diff
@@ -23,11 +23,12 @@ def get_meta_list(self, firmware_list=None):
             firmware_list = self.firmwares.find()
         for firmware in firmware_list:
             if firmware:
-                if firmware['processed_analysis']['unpacker']['file_system_flag']:
-                    unpacker = self.retrieve_analysis(deepcopy(firmware['processed_analysis']))['unpacker']['plugin_used']
+                if 'tags' in firmware:
+                    tags = firmware['tags']
                 else:
-                    unpacker = firmware['processed_analysis']['unpacker']['plugin_used']
-                list_of_firmware_data.append((firmware['_id'], self.get_hid(firmware['_id']), unpacker))
+                    tags = dict()
+                tags[firmware['processed_analysis']['unpacker']['plugin_used']] = TagColor.LIGHT_BLUE
```

Comment:
There is a bug in case the unpacking result is sanitized.

Reply:
should be fixed